snuba/datasets/configuration/generic_metrics/storages/org_counters.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+kind: readable_storage
+name: generic_org_metrics_counters
+storage:
+  key: generic_org_metrics_counters
+  set_key: generic_org_metrics_counters
```

Comment:
This should be `generic_metrics_counters`. The `set_key` is different from the `key`, and specifies which cluster the storage is referencing. You don't need a new cluster.

Reply:
fixed.